docs/specifying-a-call-to-configure.md

```diff
@@ -55,6 +55,9 @@ A.CallTo(fakeShop).Throws(new Exception());
 // Or limit the calls by return type
 A.CallTo(fakeShop).WithReturnType<string>().Returns("sugar tastes good");
 
+// Or limit the calls to methods that return a value
+A.CallTo(fakeShop).WithNonVoidReturnType().Returns("sugar tastes good");
+

```

Comment:
I think this example should go above the `WithReturnType` example, since it is less restrictive. Each example seems to be more restrictive than the one above.